Neos.Neos/Classes/Service/ContentElementWrappingService.php

```diff
@@ -64,17 +64,18 @@ class ContentElementWrappingService
      *
      * @param NodeInterface $node
      * @param string $content
-     * @param string $typoScriptPath
+     * @param string $fusionPath
      * @return string
      */
-    public function wrapContentObject(NodeInterface $node, $content, $typoScriptPath)
+    public function wrapContentObject(NodeInterface $node, $content, $fusionPath)
     {
         if ($this->needsMetadata($node, false) === false) {
             return $content;
         }
 
         $attributes = [];
-        $attributes['data-node-__typoscript-path'] = $typoScriptPath;
+        $attributes['data-node-__typoscript-path'] = $fusionPath; // @deprecated
```

Comment:
+1 In general I think it would be nice to write sth like "Deprecated since version X and will be removed with version Y" also in the code deprecations above